quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
 
```

Comment:
Hi @alastori, thank you for writing this detailed document. It is quite helpful for users to learn how to use the monitoring tools and scale clusters. I’ve tested the steps on my macOS, and while following the instructions, these steps guide me to set up a local cluster four times as follows  😂 , which is time-consuming. 

tiup playground
tiup playground --tag dev-env
tiup playground v8.1.0 --tag sample-app-env --db 2 --kv 3 --pd 1 --tiflash 0
tiup playground v8.1.0 --tag sample-app-env --db 2 --kv 3 --pd 1 --tiflash 0 --tiproxy 1

Would it be possible to provide just one or two cluster setup commands and list the rest as a tip? This could help streamline the setup process for users.

By the way, I’m not sure if introducing the usage of `--tag` is necessary for first-time users, as it makes the command longer. Without introducing `--tag` in this doc, we can avoid concerns about whether users are scaling the correct cluster.